cmd/maya-apiserver/app/command/start.go

```diff
@@ -94,6 +94,7 @@ type CmdStartOptions struct {
 	httpServer *server.HTTPServer
 }
 
+// NewCmdStart start mayaserver
```

Comment:
I think this is incorrect. This function returns a cobra command to start a server.

Reply:
Okay, I am checking for that.

Reply:
@abhishek-19 
Put it like this:
// NewCmdStart creates start command for maya-apiserver